src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/IMethodCodeNode.cs

```diff
@@ -10,6 +10,7 @@ interface IMethodCodeNode : IMethodNode, ISymbolDefinitionNode
     {
         void SetCode(ObjectNode.ObjectData data);
         void InitializeFrameInfos(FrameInfo[] frameInfos);
+        void InitializeEHClauseInfos(EHClauseInfo[] ehClauseInfos);
```

Comment:
This may be rather called DebugEHClauseInfo to make it clear it is just for debugger.

Reply:
done